packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGeneratorPage.tsx

```diff
@@ -139,7 +139,7 @@ export class ThemeGeneratorPage extends React.Component<any, IThemeGeneratorPage
           >
             <ColorPicker
               color={ colorPickerSlotRule.color!.str }
-              onColorChanged={ this._semanticSlotRuleChanged.bind(this, colorPickerSlotRule) }
+              onColorChanged={ this._getOnSemanticSlotRuleChanged(colorPickerSlotRule) }
```

Comment:
umm. Not sure this is what you meant. This will execute a function and set onColorChanged as the return value.

Reply:
ITT he's probably just removing bind. The extra time saved at render is minimal either way.

Reply:
My comment is wrong, ignore me. I have no idea.

Reply:
Yes, _getOnSemanticSlotRuleChanged returns a function.

Reply:
I need that slot though... I can move it to a private var but that's spaghetti code.

Reply:
So why bother replacing the bind with a lambda? They're the same. I'm having a hard time following the purpose of this PR. Did you actually measure a performance difference?